src/main/java/org/waarp/openr66/protocol/localhandler/packet/StartupPacket.java

```diff
@@ -31,6 +31,7 @@
  */
 public class StartupPacket extends AbstractLocalPacket {
     private final Integer localId;
+    private final boolean fromSsl;
```

Comment:
Quick question: does it breaks the backward compatibility ? Does previous versions of R66 will be able to connect to this new one, while the field is missing, and still being correct (fromSsl being false always then) ?

Reply:
StartupPacket is a LocalOnly packet send from the ServerChannel to the LocalChannel it has absolutely no impact on backward compatibility between partners.

Reply:
Perfectly right !!!